code/game/machinery/iv_drip.dm

```diff
@@ -9,6 +9,19 @@
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 	var/obj/item/reagent_containers/iv_bag/bag = null
 
+/obj/machinery/iv_drip/Initialize(mapload)
+	. = ..()
+	if(mapload)
```

Comment:
Could you make this use early returns instead?

Reply:
I changed it in the newest commit to be a late init because there was some weird stuff with overlays not properly showing up, not sure if an early return is needed or even applicable in the newest commit, a re-review would be appreciated.